src/wasm-lib/kcl/tests/import_cycle1/execution_error.snap

```diff
@@ -1,12 +1,13 @@
 ---
 source: kcl/src/simulation_tests.rs
 description: Error from executing import_cycle1.kcl
+snapshot_kind: text
 ---
 KCL ImportCycle error
 
   × import cycle: circular import of modules is not allowed: tests/
   │ import_cycle1/import_cycle2.kcl -> tests/import_cycle1/import_cycle3.kcl
-  │ -> tests/import_cycle1/input.kcl -> tests/import_cycle1/import_cycle2.kcl
+  │ -> tests/import_cycle1/input.kcl
```

Comment:
maybe the error is just not including the root path?

Reply:
i think thats it, i dont think the root path is ever added to import_stack, unsure how / where to fix

Reply:
it did fix part of the error tho

Reply:
Yes, you're right.